workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/internal/WorkflowRunner.kt

```diff
@@ -52,7 +56,7 @@ internal class WorkflowRunner<PropsT, OutputT, RenderingT>(
   private val rootNode = WorkflowNode(
     id = workflow.id(),
     workflow = workflow,
-    initialProps = currentProps,
+    initialProps = currentProps.value,
```

Comment:
Nit: I don't see you using the delegate syntax for `MutableState`s very much, it would make a lot of the code in this PR more idiomatic and a teeny bit cleaner.

Reply:
The delegate syntax continues to confuse me and I don't necessarily like the syntactic sugar obfuscating what is happening.

In this case `currentProps` is a class property. How would I use the delegate syntax here as I thought that's only when using `remember`? And `remember` is only for within a `@Composable`?

Reply:
Remember has nothing to do with delegation. Property delegation is a [well-documented Kotlin language feature](https://kotlinlang.org/docs/delegated-properties.html). It's supported for `MutableState` because it has `getValue` and `setValue` extension methods.

Reply:
Ah I was getting confused by all the examples including the `remember` in the delegate.

I see now that you can just use:
```
val currentProps: PropsT by mutableStateOf(props.value)
```

that makes a lot more sense.